apps/catalogue/src/components/HarmonizationDefinition.vue

```diff
@@ -7,17 +7,17 @@
         <dd class="col-10">
           <template
             v-if="variable.cohortMapping && variable.cohortMapping.description"
-            >{{ variable.cohortMapping.description }}</template
```

Comment:
white space removed by accident ?

Reply:
I think we should agree on formatter. I saw similar things in your pull request